packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -373,8 +389,59 @@ module.exports = (
             type: GraphQLBoolean,
             defaultValue: false,
           },
+          format: {
+            type: GraphQLString,
```

Comment:
Do we want to use an [enum type](https://graphql.org/graphql-js/type/#graphqlenumtype) here? It seems like only `plain` and `html` are options, and designating string could lead to subtle bugs (e.g. typos or something!).